onward/app/views/fragments/richLinkBody.scala.html

```diff
@@ -42,10 +43,18 @@
 
         <div class="rich-link__header">
             <h1 class="rich-link__title">
-                @if(content.content.cardStyle == Comment ||
-                    (content.content.cardStyle == SpecialReport && content.content.hasTonalHeaderByline)
-                ) {
-                    @fragments.inlineSvg("quote", "icon")
+                @if(!experiments.ActiveExperiments.isParticipating(experiments.Garnett)) {
+                    @if(content.content.cardStyle == Comment ||
+                        (content.content.cardStyle == SpecialReport && content.content.hasTonalHeaderByline)
+                    ) {
+                        @fragments.inlineSvg("quote", "icon")
+                    }
+                } else {
+                    @if(content.content.cardStyle == Comment ||
+                        (content.content.cardStyle == Feature && content.content.metadata.pillar.exists(_.toString.toLowerCase == "sport"))
```

Comment:
Don't know if there's a better way to get the pillar name? (I would've thought `Pillar` formed an ADT but apparently not)